### ISSUE TITLE
[Accessibility][Visual Requirement-Create Node]: Contrast ratio of "+" i.e. 'Add Node' button within flow chart is less than 3:1.

### ISSUE THREAD

**Issue by ashish315** [microsoft/BotFramework-Composer#2083]
**User Experience** 
Users with low vision will not be able to recognize the controls and border if non-text contrast ratio is less than 3:1.

**Test Environment**
Browser: Microsoft Edge Canary Version: 82.0.425.0 (Official build)
OS build: 2004 (19564.1005)
Screen Reader: Narrator
URL: [Bot Framework Composer](http://localhost:3000/dialogs/Main)
[**Prerequisite Here**](https://github.com/microsoft/BotFramework-Composer)

**Repro Steps:** 
1. Open the above mentioned URL in EDGE and set the zoom at 400%.
2. Select 'Design Flow' from left navigation.
3. Now, navigate to the flow chart.
4. Navigate to "+" button and activate it.
5. Observe the CCA with color contrast analyzer and find the issue.

**Actual:** 
The color contrast ratio of + button is 2.3:1 which is less than 3:1.

**Expected:**
The color contrast of + button should be more than or equals to 3:1. 

**Observation:**
CCA of arrow is also very low i.e. 2.3:1. Its CCA should be equal or more than 3:1 as it symbolize the flow of the nodes.

**MAS Impacted:** [MAS1.4.11](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/Doc.aspx?sourcedoc=%7B6CA3BF14-2635-434A-832C-28DD9A18B9FC%7D&file=MAS%201.4.11%20%E2%80%93%20Non-text%20Contrast.docx&action=default&mobileredirect=true&cid=49a9d490-290c-41e7-bc24-427e0c15a53e)

**Attachment:**
![MAS1 4 11_CCA of Add node button](https://user-images.githubusercontent.com/60913131/75252420-a8990180-5802-11ea-919c-549658a579d4.png)
![MAS1 4 11_CCA of Arrow's](https://user-images.githubusercontent.com/60913131/75252743-455b9f00-5803-11ea-8fdd-389e799d990c.png)

**Comment by DesignPolice** [comment]
Thanks @ashish315 

@cwhitten when we do the next move to Stable a few of the UI changes will change on this - 

1) the visual canvas will be lighter - It was changed from E3E3E3 to F6F6F6 - so that will help.

Gray arrows should be -  656565 on the F6F6F6 will land us at 5.39:1

+ node is white with a Depth 2 shadow and the blue "plus" is now a bit larger with #0078D4 Brand color so that lands at 4.5, which isn't ideal, but not sure what to do with a brand color that is made to barely pass. 

![Screen Shot 2020-02-25 at 5 11 30 PM](https://user-images.githubusercontent.com/45223620/75302244-0ffa8580-57f2-11ea-84a3-6a048ba41404.png)

**Comment by corinagum** [comment]
@DesignPolice Could you also include what the focus contrast ratios will be? :)

**Comment by DesignPolice** [comment]
@corinagum update on these focus colors. Talking to Marieke if we change the blue on the visual canvas, we would have to change everywhere in the app, to not have a strange two color blue for the select state. 

So let's see if we can continue to use the same blue on the canvas - which is still AA at 3.63

**Comment by yeze322** [comment]
the visual canvas will be lighter - It was changed from E3E3E3 to F6F6F6 - so that will help.
Gray arrows should be - 656565 on the F6F6F6 will land us at 5.39:1

**Comment by alanlong9278** [comment]
So, do we need to change the `+` and `arrow` node color to update the contrast ratio? 
- the `+` color is `#0078D4`,
- the `arrow ` color is `#545454`.

And what color should we set? 
@DesignPolice

**Comment by DesignPolice** [comment]
Hi @alanlong9278 -

We should be fine with the + in the #0078D4 - the circle behind it is white - and had a drop shadow.

The lines and arrows should all be the same colors with the #656565 as @yeze322 mentioned above, thanks for checking in - 

Appreciate all the hard work on this. M

**Comment by alanlong9278** [comment]
Fine, Thanks

**Comment by ashish315** [comment]
As checked and verified on below environment, the issue is fixed now. So, we are closing the issue.

**Test Environment**
Browser: Microsoft Edge Dev {Version 83.0.467.0 (Official build) dev (64-bit)}
OS build: 2004 (19588.1000)
Screen Reader: Narrator
URL: [Bot Framework Composer](http://localhost:3000/dialogs/Main)
[**Prerequisite Here**](https://github.com/microsoft/BotFramework-Composer)
